$NetBSD: patch-ac,v 1.5 2013/03/01 14:33:51 gdt Exp $

Leftover hunk from "Fix build with png-1.6."  (Most were applied upstream.)

--- libs/auxfun/image.cpp.orig	2012-05-22 05:55:37.000000000 -0400
+++ libs/auxfun/image.cpp	2013-03-01 09:14:37.000000000 -0500
@@ -7,6 +7,7 @@
 #include <zlib.h>
 #include <png.h>
 #include <stdlib.h>
+#include <string.h>
 #ifdef _MSC_VER
 #define _USE_MATH_DEFINES 1
 #include <math.h>
